scripts/demo-projects.android.sh

```diff
@@ -38,8 +38,3 @@ popd
 pushd detox
   run_f "npm run build:android-native"
 popd
-
-pushd examples/demo-pure-native-android
-  export ANDROID_SERIAL=`adb devices | grep emulator | head -1 | awk '{print $1}'`
-  run_f "./gradlew connectAndroidTest"
-popd
```

Comment:
Can't we keep that?

Reply:
Only if you want me to put a few more hours/day on it?